### PR TITLE
[6/11] Implements Page header in Networks, Cluster and Repositories pages

### PR DESCRIPTION
**Pull request by rcorral**
---
~~⚠️   _This branch depends on features that will be introduced with #1482_~~ Merged!

---

### CODE REVIEW

**Comment by rcorral** [comment]
retest this please

**Comment by orlandohohmeier** [comment]
@mlunoe why did you close this PR?

File: src/js/pages/system/RepositoriesTab.js [hunk 1]
```diff
@@ -83,37 +93,41 @@ class RepositoriesTab extends mixin(StoreMixin) {
...
-    if (hasError) {
-      return this.getErrorScreen();
-    }
+    let content = null;
 
-    if (isLoading) {
-      return this.getLoadingScreen();
+    if (hasError) {
+      content = this.getErrorScreen();
+    } else if (isLoading) {
```
Comment (mlunoe) [inline src/js/pages/system/RepositoriesTab.js]: let's create a separate function instead of these `if-else` statements

Comment (mlunoe) [inline src/js/pages/system/RepositoriesTab.js]: Done!

**Comment by rcorral** [comment]
retest this please

File: src/js/pages/network/VirtualNetworksTab.js [hunk 2]
```diff
@@ -76,7 +86,9 @@ class VirtualNetworksTabContent extends mixin(StoreMixin) {
   }
 
   getErrorScreen() {
-    return <RequestErrorMsg />;
+    return (
```
Comment (MatApple) [inline src/js/pages/network/VirtualNetworksTab.js]: We don't need these extra parenthesis

File: src/js/pages/network/VirtualNetworksTab.js [hunk 3]
```diff
@@ -91,7 +103,9 @@ class VirtualNetworksTabContent extends mixin(StoreMixin) {
   }
 
   getLoadingScreen() {
-    return <Loader />;
+    return (
```
Comment (MatApple) [inline src/js/pages/network/VirtualNetworksTab.js]: Don't need this either

File: src/js/pages/network/virtual-network-detail/VirtualNetworkDetail.js [hunk 4]
```diff
@@ -73,53 +86,61 @@ class VirtualNetworkDetail extends mixin(StoreMixin, TabsMixin) {
...
+      </Page>
+    );
   }
 
   render() {
     let {errorCount, receivedVirtualNetworks} = this.state;
     if (errorCount >= 3) {
-      return this.getErrorScreen();
+      return (
+        <Page>
```
Comment (MatApple) [inline src/js/pages/network/virtual-network-detail/VirtualNetworkDetail.js]: This should be `return this.getErrorScreen();`

File: src/js/pages/network/VirtualNetworksTab.js (hunk 2 above)
Comment (jfurrow) [inline src/js/pages/network/VirtualNetworksTab.js]: Done!

File: src/js/pages/network/VirtualNetworksTab.js (hunk 3 above)
Comment (jfurrow) [inline src/js/pages/network/VirtualNetworksTab.js]: Done!

File: src/js/pages/network/virtual-network-detail/VirtualNetworkDetail.js (hunk 4 above)
Comment (jfurrow) [inline src/js/pages/network/virtual-network-detail/VirtualNetworkDetail.js]: Done!

**Comment by mlunoe** [comment]
retest this please

File: src/js/pages/NetworkPage.js [hunk 5]
```diff
@@ -8,37 +8,20 @@ import {Hooks} from 'PluginSDK';
...
-  '/networking/networks': {
-    content: 'Networks',
-    priority: 20
-  }
-};
-
-const DEFAULT_NETWORK_SUB_TABS = {};
+const NetworkingBreadcrumbs = () => {
+  const crumbs = [
+    <Link to="networking" key={-1}>Networks</Link>
```
Comment (orlandohohmeier) [inline src/js/pages/NetworkPage.js]: The negative key looks rather funky. Please use a different unique identifier, if possible. 

🗒  _Side Note: We should always prefer more specific keys over indexes to not only make the validator happy but actually improve the rendering performance._

**Comment by orlandohohmeier** [comment]
🚢🚢🚢

**Pull request merged**